packages/MJStorage/src/drivers/GoogleFileStorage.ts

```diff
@@ -0,0 +1,56 @@
+import { Storage } from '@google-cloud/storage';
```

Comment:
@cadam11 so with these drivers there is only a server side? The client side is the same for all of them? If we had a future scenario where another client required slightly different handling - making this up but say box.com required something different - can we handle or do we need more infrastructure for that bit?

Reply:
I was expecting that they'd all have slight differences on the FE but it turned out they were all super standard so no need for special handling on the client. Since this covers the big 3 as part of the core module, I think other providers we'd ship as separate packages (e.g. `@memberjunction/storage-box`) that would export both the server component (the driver file) and a plugin for the front end. At that point we'd add a socket of some kind on the client to easily plug in the custom client behavior.

Reply:
@cadam11  got it. I think at that point we could define some client-side base class to handle this and use our usual technique of RegisterClass to abstract the instantiation of the actual driver class to avoid close-coupling, but those are details we can do if we need to down the road. Excited to have the "big 3" done!